infrastructure/models.py

```diff
@@ -140,6 +179,51 @@ def fuzzy_commencement_date(self):
     def fuzzy_planned_completion_date(self):
         return fuzzydate(self.planned_completion_year, self.planned_completion_month, self.planned_completion_day)
 
+    construction_start_year = models.PositiveSmallIntegerField(blank=True, null=True)
+    construction_start_month = models.PositiveSmallIntegerField(blank=True, null=True)
+    construction_start_day = models.PositiveSmallIntegerField(blank=True, null=True)
+    
+    @property
+    def fuzzy_construction_date(self):
+        return fuzzydate(self.construction_start_year, self.construction_start_month, self.construction_start_day)
+    
+    project_output = models.BigIntegerField(
+        blank=True, null=True,
+    )
+
+    project_output_unit = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectPlantUnits.UNITS
+    )
+
+    project_output_year = models.PositiveSmallIntegerField(blank=True, null=True)
+    @property
+    def fuzzy_output_date(self):
+        return fuzzydate(self.project_output_year)
+
+    estimated_project_output = models.BigIntegerField(
+        blank=True, null=True,
+    )
+    estimated_project_unit = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectPlantUnits.UNITS
+    )
+    project_capacity = models.BigIntegerField(
```

Comment:
In the CSV file this field oftentimes has a decimal in it. Using a `FloatField` should be able to get that data.